### ISSUE TITLE
some local cucumber failures

### ISSUE THREAD

**Issue by tansaku** [AgileVentures/WebsiteOne#751]
was checking @alejandrobabio 's rm ytube branch locally and I get these failures in cucumber:

```
(::) failed steps (::)

Unable to find css "#liveHOA-placeholder iframe" (Capybara::ElementNotFound)
./features/step_definitions/hangout_steps.rb:2:in `/^I should see hangout button$/'
features/hangout.feature:27:in `Then I should see hangout button'

Unable to find css "#liveHOA-placeholder iframe" (Capybara::ElementNotFound)
./features/step_definitions/hangout_steps.rb:2:in `/^I should see hangout button$/'
features/hangout.feature:103:in `Then I should see hangout button'

expected to find text "00:00 15/04" in "Anders Persson My account Log out About Projects Members Articles Events Upcoming events Past scrums Past events Getting started Current Projects hello world hello mars Supporters Become a supporter Projects hello world hello world greetings earthlings Start Live Hangout × Info You should join this project before you can start the hangouts Close Project status: ACTIVE Created less than a minute ago by Tester Person Join Project Connected on not linked to GitHub not linked to PivotalTracker Members (0) No members in project hello world × The hello world team About Documents (0) Videos (3) Activity (0) Project content missing :( A compelling pitch can make your project more appealing to potential collaborators. Please edit project details to add pitch content. No documents can be found for project hello world PP on hello world - feature: 2 Latest Project videos Video Host Published PP on hello world - feature: 2 John Doe 00:02 14/04 PP on hello world - feature: 1 John Doe 00:01 14/04 PP on hello world - feature: 0 John Doe 00:00 14/04 No PivotalTracker Stories can be found for project hello world Learn More About Us Getting Started Dashboard Social Facebook Twitter Our Sponsors Makers Academy AirPair Becoming a sponsor AgileVentures - Crowdsourced Learning Contact us Send a traditional email to info@agileventures.org." (RSpec::Expectations::ExpectationNotMetError)
./features/step_definitions/basic_steps.rb:161:in `block (2 levels) in <top (required)>'
./features/step_definitions/basic_steps.rb:160:in `each'
./features/step_definitions/basic_steps.rb:160:in `/^I should( not)? see:$/'
features/project_videos.feature:24:in `And I should see:'

Failing Scenarios:
cucumber features/hangout.feature:25 # Scenario: Create a hangout for a scrum event
cucumber features/hangout.feature:100 # Scenario: Display hangout button on a project's page
cucumber features/project_videos.feature:16 # Scenario: Project show page renders a list of videos
```

CI is fine - so assume I am missing something on my local system ...

**Comment by tansaku** [comment]
@alejandrobabio @diraulo do either of you ever see the above locally?

**Comment by alejandrobabio** [comment]
@tansaku I had some issues with capybara at my local system, which I could fix adding some delay (sleep 0.5). Today I buy a SSD drive and I expect it fixes the issues after I'll install it.

Another thing I will try, is add a `save_and_open_page` at `features/step_definitions/hangout_steps.rb:2` before the find.

I hope it helps!
